bridges/CubariProxyBridge.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+class CubariProxyBridge extends BridgeAbstract
+{
+    const NAME = 'Cubari Proxy';
+    const MAINTAINER = 'phantop';
+    const URI = 'https://cubari.moe';
+    const DESCRIPTION = 'Returns chapters from Cubari.';
+    const PARAMETERS = [[
+        'service' => [
+            'name' => 'Content service',
+            'type' => 'list',
+            'defaultValue' => 'mangadex',
+            'values' => [
+                'MangAventure' => 'mangadventure',
+                'MangaDex' => 'mangadex',
+                'MangaKatana' => 'mangakatana',
+                'MangaSee' => 'mangasee',
+            ]
+        ],
+        'series' => [
+            'name' => 'Series ID/Name',
+            'exampleValue' => '8c1d7d0c-e0b7-4170-941d-29f652c3c19d', # KnH
+            'required' => true,
+        ],
+        'fetch' => [
+            'name' => 'Fetch chapter page images',
+            'type' => 'list',
+            'title' => 'Places chapter images in feed contents. Entries will consume more bandwidth.',
+            'defaultValue' => 'c',
+            'values' => [
+                'None' => 'n',
+                'Content' => 'c',
+                'Enclosure' => 'e'
+            ]
+        ],
+    ]];
+
+    private $title;
+
+    public function collectData()
+    {
+        $url = parent::getURI() . '/read/api/' . $this->getInput('service') . '/series/' . $this->getInput('series');
+        $json = Json::decode(getContents($url));
+        $this->title = $json['title'];
+
+        $chapters = $json['chapters'];
+        krsort($chapters);
+
+        foreach ($chapters as $number => $element) {
+            $item = [];
+            $item['uri'] = $this->getURI() . '/' . $number;
+
+            if ($element['title']) {
+                $item['title'] = $number . ' - ' . $element['title'];
+            } else {
+                $item['title'] = 'Volume ' . $element['volume'] . ' Chapter ' . $number;
+            }
+
+            $group = '1';
+            if (array_key_exists('release_date', $element)) {
+                $dates = $element['release_date'];
+                $date = max($dates);
+                $item['timestamp'] = $date;
+                $group = array_keys($dates, $date)[0];
+            }
+            $page = $element['groups'][$group];
+            $item['author'] = $json['groups'][$group];
+            $api = parent::getURI() . $page;
+            $item['uid'] = $page;
+            $item['comments'] = $api;
```

Comment:
the `comments` property is currently not used by atom and mrss. but maybe in the future it will :)

Reply:
I'm aware! My own, personal instance has small changes to make the comments property useable and I'm preemptively adding it to any bridges I submit (if applicable) for the event that it gets added.